src/common/components/repository-row/index.js

```diff
@@ -354,31 +357,29 @@ class RepositoryRow extends Component {
 
 RepositoryRow.propTypes = {
   snap: PropTypes.shape({
-    resource_type_link: PropTypes.string,
     git_repository_url: PropTypes.string,
-    self_link: PropTypes.string,
     store_name: PropTypes.string,
     snapcraft_data: PropTypes.object
   }),
   latestBuild: PropTypes.shape({
     buildId: PropTypes.string,
-    status: PropTypes.string,
+    buildLogUrl: PropTypes.string,
+    colour: PropTypes.string,
```

Comment:
Cleaning up in `PropTypes`. I think only properties that are used by given component should be listed here.